tests/test_model.py

```diff
@@ -262,7 +262,9 @@ def test_component_model():
 
 def test_container_purl():
     # TODO: Failed due to "assert arch not in purl"
-    container = ContainerImageComponentFactory()
+    # With the changes introduced in CORGI-678 the container name should be equal to the last part
+    # of the repository_url
+    container = ContainerImageComponentFactory(name="node-exporter-rhel8")
```

Comment:
To make this a little more clear, I would do below:
```suggestion
    repo_name = "node-exporter-rhel8"
    # With the changes introduced in CORGI-678 the container name should be equal to the last part
    # of the repository_url
    container = ContainerImageComponentFactory(name=repo_name)
```
And then remove the repo_name redefinition from the old line 275 / new line 277 below.

Reply:
good idea